.github/workflows/ci.yml

```diff
@@ -41,3 +28,41 @@ jobs:
         make
         sudo make install
         ctest
+
+  build_windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        arch:
+          - { mingw: "64", msys: x86_64, python: "x64"}
+        python_version:
+          - 3.6
+          - 3.9
+        mgversion:
+          - 1.4.0
```

Comment:
first, you don't use python_version or mgversion, so it can be removed
second, you don't need matrix or variables even, you can hardcode arch stuff for now.